test/cli-parser/handler.spec.js

```diff
@@ -39,6 +39,37 @@ describe('CLI parser', () => {
       assert.equal(compiler.lastArgs[1].plugins.length, 0)
     })
 
+    it('should use first non-hypenated option as source folder if no --source is defined', () => {
+      compiler.reset()
```

Comment:
Yep we should, my problem is that I'm to used to tape/Ave abd I really dislike before/after hooks from mocha 😄